docs/_data/nav.yml

```diff
@@ -8,6 +8,8 @@
       path: index.html#usecases
     - name: Features
       path: index.html#opengeefeatures
+    - name: Release 5.2.0
+      path: https://github.com/google/earthenterprise/releases/tag/5.2.0-3.final
```

Comment:
This link doesn't work - it goes to https://tst-lsavoie.github.io/earthenterprise/https://github.com/google/earthenterprise/releases/tag/5.2.0-3.final.  There is probably some way to indicate that it is an absolute URL, not a relative one.

Reply:
By the way, I pushed these changes up to my fork, so you can see them at https://tst-lsavoie.github.io/earthenterprise.